sim/paladin/talents.go

```diff
@@ -49,7 +49,7 @@ func (paladin *Paladin) getIlluminationActionID() core.ActionID {
 	}
 }
 
-func (paladin *Paladin) getBonusCritChanceFromHolyPower() float64 {
+func (paladin *Paladin) holyPower() float64 {
 	return core.CritRatingPerCritChance * float64(paladin.Talents.HolyPower)
 }
```

Comment:
🚫 This change takes information away about what this method is doing. Longer, more descriptive method names are better than shorter ones that don't convey the proper intention

Reply:
This method is only used to set "BonusCritRating", so I think the longer name is rather redundant. But I'll change to house style.

Reply:
Thanks 🙏 it just adds clarity for those who aren't as familiar with the class's mechanics